packages/semcom-components/lib/demo.ts

```diff
@@ -38,8 +38,20 @@ document.addEventListener(ComponentEventType.WRITE, (event: ComponentWriteEvent)
 
   }
 
-  setTimeout(() => event.target?.dispatchEvent(new ComponentResponseEvent({
-    detail: { ...event.detail, cause: event, success: true },
-  })), 2000);
+  try{
+
+    new URL('http://' + event.detail.uri + '.net');
```

Comment:
Why do we need to add the scheme and top-level domain? This way, even strings like `foo bar` are valid inputs for the component. I would just use the uri here.